include/velocypack/Slice.h

```diff
@@ -710,13 +721,19 @@ class Slice {
 
   template<typename T>
   T getNumber() const {
-    if (std::is_integral<T>()) {
```

Comment:
hm, it works because caused ctor of std::is_integral and std::is_integral is derived class for class which can convert to bool